api/src/routes/user.ts

```diff
@@ -301,5 +303,59 @@ export const userRoutes: FastifyPluginCallbackTypebox = (
     }
   );
 
+  fastify.post(
+    '/user/report-user',
+    {
+      schema: schemas.reportUser,
+      preValidation: (req, reply, done) => {
+        // NOTE: Is this the right hook?
+        req.body = {
+          ...req.body,
+          reportDescription: trimTags(req.body.reportDescription)
+        };
```

Comment:
```suggestion
        req.body.reportDescription = trimTags(req.body.reportDescription);
```
Since we have no use for the old body, I don't see the harm in mutating it here.